plugins/modules/ios_ntp_global.py

```diff
@@ -0,0 +1,1088 @@
+# ntp authentication-key 2 md5 SomeSecurePassword 22
+# ntp peer 172.16.1.10 version 2
+# ntp peer 172.16.1.11 key 2 minpoll 5 prefer version 2
+# ntp peer ip checkPeerDomainIpv4.com prefer
+# ntp peer ipv6 checkPeerDomainIpv6.com
+# ntp peer ipv6 testPeerDomainIpv6.com prefer
+# ntp server 172.16.1.12 version 2
+# ntp server ipv6 checkServerDomainIpv6.com
+# ntp server 172.16.1.13 source GigabitEthernet0/1
+# ntp trusted-key 3 - 13
+# ntp trusted-key 21
+
+# Parsed play:
+# ------------
+
+- name: Parse the provided configuration with the existing running configuration
+  cisco.ios.ios_ntp_global:
+    running_config: "{{ lookup('file', 'parsed.cfg') }}"
+    state: parsed
+
+# Module Execution Result:
+# ------------------------
+
+# "parsed": {
+#     "access_group": {
+#         "peer": [
+#             {
+#                 "access_list": "2",
+#                 "kod": true
+#             },
+#             {
+#                 "access_list": "DHCP-Server",
+#                 "ipv4": true,
+#                 "kod": true
+#             },
+#             {
+#                 "access_list": "preauth_ipv6_acl",
+#                 "ipv6": true,
+#                 "kod": true
+#             }
+#         ],
+#         "query_only": [
+#             {
+#                 "access_list": "10"
+#             }
+#         ]
+#     },
+#     "allow": {
+#         "control": {
+#             "rate_limit": 4
+#         },
+#         "private": true
+#     },
+#     "authenticate": true,
+#     "authentication_keys": [
+#         {
+#             "algorithm": "md5",
+#             "encryption": 22,
+#             "id": 2,
+#             "key": "SomeSecurePassword"
+#         }
+#     ],
+#     "broadcast_delay": 22,
+#     "clock_period": 5,
+#     "logging": true,
+#     "master": {
+#         "stratum_number": 4
+#     },
+#     "max_associations": 34,
+#     "max_distance": 3,
+#     "min_distance": 10,
+#     "orphan": 4,
+#     "panic_update": true,
+#     "peers": [
+#         {
+#             "peer": "172.16.1.10",
+#             "version": 2
+#         },
+#         {
+#             "peer": "checkPeerDomainIpv6.com",
+#             "use_ipv6": true
+#         }
+#     ],
+#     "servers": [
+#         {
+#             "server": "172.16.1.12",
+#             "version": 2
+#         },
+#         {
+#             "server": "172.16.1.13",
+#             "source": "GigabitEthernet0/1"
+#         },
+#         {
+#             "server": "checkServerDomainIpv6.com",
+#             "use_ipv6": true
+#         }
+#     ],
+#     "source": "GigabitEthernet0/1",
+#     "trusted_keys": [
+#         {
+#             "range_start": 21
+#         },
+#         {
+#             "range_end": 13,
+#             "range_start": 3
+#         }
+#     ],
+#     "update_calendar": true
+# }
+"""
+
+RETURN = """
+before:
+  description: The configuration prior to the module execution.
+  returned: when state is I(merged), I(replaced), I(overridden), I(deleted) or I(purged)
```

Comment:
These have been updated in latest builder about ~20 days back. Please have a look at the template.